### DISCUSSION TITLE
DynamicLinks and prefetch

### DISCUSSION THREAD

**Discussion by rpominov** [sergiodxa/remix-utils#112] [Q&A] [answered]
Hi there! 

Does `<DynamicLinks>` support prefetch the same way `<Links>` do? 

I assume it doesn't, but this isn't explicitly stated in the documentation, so I wanted to ask.

And, if it doesn't, is it completely impossible to have prefetch work along with having at least the request parameters in the `dynamicLinks` function? Maybe there were some attempts to make this work, or some future plans?

**Comment by sergiodxa** [comment, marked as answer]
It doesn't, there's no way to attach things to what `<Link prefetch>` prefetch and also because DynamicLinks depends on the data of the route and Remix doesn't manually fetch that when prefetching and instead rely on `<link rel="prefetch">` tags then there's no way to call the DynamicLinksFunction and know the complete list of links.
